CHANGELOG.md

```diff
@@ -12,3 +22,7 @@
   * adds FlattenJSON as default adapter [@joaomdmoura]
```

Comment:
There should be a note somewhere stating that the FlattenJSON adapter changed name.